Meadowlark-js/packages/meadowlark-authz-server/test/security/TokenValidator.test.ts

```diff
@@ -167,23 +167,13 @@ describe('given introspection of expired token', () => {
   });
 
   it('returns expired', () => {
-    expect(result).toMatchInlineSnapshot(`
-      {
-        "introspectedToken": {
-          "active": false,
-          "aud": "ed-fi-meadowlark",
-          "client_id": "clientId",
-          "exp": 1666195391,
-          "iat": 1666198991,
-          "iss": "ed-fi-meadowlark",
-          "roles": [
-            "vendor",
-          ],
-          "sub": "",
-        },
-        "isValid": true,
-      }
-    `);
+    expect(result.isValid).toBe(true);
+    expect((result as any).introspectedToken.active).toBe(false);
+    expect((result as any).introspectedToken.aud).toBe(TOKEN_ISSUER);
+    expect((result as any).introspectedToken.client_id).toBe('clientId');
+    expect((result as any).introspectedToken.iss).toBe(TOKEN_ISSUER);
+    expect((result as any).introspectedToken.roles).toHaveLength(1);
+    expect((result as any).introspectedToken.roles[0]).toBe('vendor');
```

Comment:
I don't have any concern with this, as I am not used to snapshots anyway. Just curious: why change styles?

Reply:
Because the token includes timestamps. Always a surprise when the snapshot looks good at first but won't pass the second time. 🤣